frontend/src/metabase/public/components/widgets/SharingPane/SharingPaneButton/SharingPaneButton.styled.tsx

```diff
@@ -0,0 +1,63 @@
+import styled from "@emotion/styled";
+import type { HTMLAttributes } from "react";
+import { css } from "@emotion/react";
+import type { PaperProps, ButtonProps } from "metabase/ui";
+import { Button, Paper, Title } from "metabase/ui";
+import { StaticEmbedIcon } from "../icons";
+
+export const SharingPaneButtonTitle = styled(Title)``;
+export const SharingPaneActionButton = styled(Button)<
+  ButtonProps & HTMLAttributes<HTMLButtonElement>
+>``;
+
+export const StaticEmbedIconWrapper = styled(StaticEmbedIcon)``;
```

Comment:
I've prototyped dealing with empty styled tags in this commit 35f9d79b464f9847e5c4dcc09cb1eba1b3d03000.

Basically, I inverse the CSS selector by putting the container CSS selector in the child component instead of the other way around.

The pros in my opinion would be we group related styles in a separate place, so there's no one big file that contains many individual children components with overridden styles.

Please take a look and if you think the approach looks good you can apply that to other empty-styled tags.

Reply:
Ah ok I see what you did. My only reservation about doing this is that I like having all of the hover states for a container to be in one place - for example, with the previous approach, I could go to `SharingPaneButtonContent` and see that the button would change colors along with everything else. But I'll take this over the previous approach still

Reply:
Fair enough, I couldn't find a way to do that without leaving empty tags. If you could do that it's fine. But I don't think having empty styled tags is a good idea. Looks like we're just hacking it together.

Reply:
I agree, I hated having them there as well which is why at the end of the day I think your approach is still better, but it is what it is